Dockerfile.base

```diff
@@ -1,33 +1,30 @@
-FROM node:14-buster
-
-ENV PATH="/root/.rbenv/shims:${PATH}"
+FROM ruby:3
 
 # Install dependencies
 RUN apt-get update \
-    && apt-get -y install sudo jq rbenv \
-    && mkdir -p "$(rbenv root)"/plugins \
-    && git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build \
-    && git -C "$(rbenv root)"/plugins/ruby-build pull \
-    && rbenv install 2.7.1 \
-    && rbenv global 2.7.1 \
-    && gem install shopify-cli -N
+  && apt-get -y install sudo jq chromium
```

Comment:
3 months ago, when I tried building the Docker image on my M1 Mac, `apt-get install -y google-chrome-stable` failed → [Google Chrome is only packaged for amd64, and is not available for ARM](https://askubuntu.com/questions/1383789/install-chrome-on-ubuntu-debian-with-arm64).

Therefore I installed `chromium`.

Due to the error _The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested_ and its fix with `--platform linux/amd64`, we might be able to use Chrome again, but I think that Chromium is fine too.